lib/aruba/api/command.rb

```diff
@@ -234,13 +234,13 @@ def run_command(*args)
       # @param [Hash] options
       #   Options for aruba
       #
-      # @option [TrueClass,FalseClass] fail_on_error
+      # @option options [TrueClass,FalseClass] fail_on_error
```

Comment:
Does it makes sense to have `Boolean`  here as well?

Reply:
There are many more files. Would you like to change it yourself? And change all occurences of `[TrueClass, FalseClass]` to `[Boolean]`

* lib/aruba/api/command.rb
* lib/aruba/api/deprecated.rb
* lib/aruba/matchers/command/be_successfully_executed.rb
* lib/aruba/matchers/command/have_exit_status.rb
* lib/aruba/matchers/command/have_finished_in_time.rb
* lib/aruba/matchers/command/have_output.rb
* lib/aruba/matchers/command/have_output_on_stderr.rb
* lib/aruba/matchers/command/have_output_on_stdout.rb
* lib/aruba/matchers/command/have_output_size.rb
* lib/aruba/matchers/directory/be_an_existing_directory.rb
* lib/aruba/matchers/directory/have_sub_directory.rb
* lib/aruba/matchers/file/be_a_command_found_in_path.rb
* lib/aruba/matchers/file/be_an_existing_executable.rb
* lib/aruba/matchers/file/be_an_existing_file.rb
* lib/aruba/matchers/file/have_file_content.rb
* lib/aruba/matchers/file/have_file_size.rb
* lib/aruba/matchers/file/have_same_file_content.rb
* lib/aruba/matchers/path/a_path_matching_pattern.rb
* lib/aruba/matchers/path/be_an_absolute_path.rb
* lib/aruba/matchers/path/be_an_existing_path.rb
* lib/aruba/matchers/path/have_permissions.rb
* lib/aruba/matchers/path/match_path_pattern.rb
* lib/aruba/matchers/rspec_matcher_include_regexp.rb
* lib/aruba/platforms/aruba_file_creator.rb
* lib/aruba/platforms/aruba_fixed_size_file_creator.rb
* lib/aruba/platforms/unix_platform.rb

Reply:
I made the replacement.

Reply:
You missed one in lib/aruba/api/command.rb. It's without an " " in between.